src/hyp_map.rs

```diff
@@ -585,7 +585,11 @@ impl HypMap {
     }
 
     /// Creates a new page table based on this memory map.
-    pub fn new_page_table(&self, hyp_mem: &mut HypPageAlloc) -> HypPageTable {
+    pub fn new_page_table(
```

Comment:
nit: This commit message needs s/launch/main/ now that that nomenclature is used.